### PR TITLE
Lab: Create and fund account

### PR DESCRIPTION
**Pull request by quietbits**

### CODE REVIEW

File: docs/tools/developer-tools/lab/account.mdx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+If you already have a keypair you want to fund, go to the "Fund Account" page.
+
+1. If you have generated a keypair on the "Create Account Keypair" page, you can use its public key by clicking the "Fill in the generated key" button. The button will be disabled if there is no keypair.
+2. You can always manually input a public key.
+3. Once you enter the public key, click the "Get lumens" button to fund the account with 10,000 XLM on the Testnet or Futurenet network.
+4. You will see a response message once the operation is completed.
+
+:::info
+
+You can also create an account using a Stellar SDK. Follow a guide [here](/docs/build/guides/basics/create-account).
```
Comment (janewang) [inline docs/tools/developer-tools/lab/account.mdx:39]: Elliot always tells me to use relative links...

**Comment by janewang** [comment]
Thank you ordering the images in the static folder! Way better, thank you! Otherwise LGTM

File: docs/tools/developer-tools/lab/account.mdx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+Anyone with the account’s secret key has full access and control over the account. Keep the secret key safe.
+
+:::
+
+3. An account is active only once it is funded. You can use Friendbot to fund it on Testnet and Futurenet networks only. Click on the "Fund account with Friendbot" button to add 10,000 XLM to the account. When the operation completes, a message will be displayed at the bottom.
+
+:::info
+
+The Friendbot can be used only on inactive accounts (accounts that don’t have any balance yet). Click the "Fund account with Friendbot" button again to see what happens. It’s the Lab, so feel free to experiment (as long as you are on Testnet or Futurenet network)!
```
[the remark this answers is not in the document]
Comment (quietbits) [reply docs/tools/developer-tools/lab/account.mdx:22]: Will we update the web version as well? I'm getting the error now on the Lab beta URL.

image

**Comment by janewang** [comment]
I added an update to the friendbot message

**Review by janewang — APPROVED** [review]

**Pull request merged**